src/main/java/sorald/cli/Cli.java

```diff
@@ -69,25 +71,32 @@ public Integer call() {
                 required = true)
         File originalFilesPath;
 
-        @CommandLine.Option(
-                names = {Constants.ARG_RULE_KEYS},
-                description =
-                        "Choose one or more of the following rule keys "
-                                + "(use ',' to separate multiple keys):\n"
-                                + Processors.RULE_DESCRIPTIONS,
-                required = true,
-                split = ",")
-        private void setRuleKeys(List<Integer> value) {
-            for (Integer ruleKey : value) {
-                if (Processors.getProcessor(ruleKey) == null) {
-                    throw new CommandLine.ParameterException(
-                            spec.commandLine(),
-                            "Sorry, repair not available for rule "
-                                    + ruleKey
-                                    + ". See the available rules below.");
-                }
-            }
-            ruleKeys = value;
+        @CommandLine.ArgGroup(multiplicity = "1")
+        Rules rules;
+
+        static class Rules {
+            @CommandLine.Option(
+                    names = {Constants.ARG_RULE_KEYS},
+                    description =
+                            "Choose one or more of the following rule keys "
+                                    + "(use ',' to separate multiple keys):\n"
+                                    + Processors.RULE_DESCRIPTIONS,
+                    required = true,
+                    split = ",")
+            List<Integer> ruleKeys;
+
+            @CommandLine.Option(
+                    names = Constants.ARG_RULE_VIOLATION_SPECIFIERS,
+                    description =
+                            "One or more rule violation specifiers. Specifiers can be gathered "
+                                    + "with the '"
+                                    + Constants.MINE_COMMAND_NAME
+                                    + "' command using the "
+                                    + Constants.ARG_STATS_OUTPUT_FILE
+                                    + " option.",
+                    required = true,
+                    split = ",")
+            List<String> ruleViolationSpecifiers;
```

Comment:
FYI this makes the options ARG_RULE_KEYS and ARG_RULE_VIOLATION_SPECIFIERS mutually exclusive in the CLI